enhancements/ingress/configurable-dns-loadbalancerservice.md

```diff
@@ -0,0 +1,199 @@
+---
+title: configurable-dns-for-loadbalancerservice
+authors:
+  - "@thejasn"
+reviewers:
+  - "@alebedev87"
+  - "@sherine-k"
+approvers:
+  - TBD
+api-approvers:
+  - TBD
+creation-date: 2022-05-10
+last-updated: 2022-05-10
+tracking-link:
+  - https://issues.redhat.com/browse/CFEPLAN-58
+see-also:
+  - N/A
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+
+# Configurable DNS Management for LoadBalancerService Ingress Controllers
+
+## Summary
+
+The ingress operator currently assumes that it's going to create and
+manage DNS using the required cloud provider integration, but in some
+circumstances when the DNS zone may be different from the cluster DNS zone
+and may not even hosted in the cloud provider. In such scenarios the operator
+after provisioning the required resources(load balancer, haproxy) in the cloud
```

Comment:
`haproxy` is not provisioned in the cloud provider, it's a deployment in OpenShift.